src/objectEnumerationBuilder.ts

```diff
@@ -24,166 +24,166 @@
  *  THE SOFTWARE.
  */
 
-module powerbi.extensibility.visual {
-
-    /**
-     * A helper class for building a VisualObjectInstanceEnumerationObject:
-     * - Allows call chaining (e.g., builder.pushInstance({...}).pushInstance({...})
-     * - Allows creating of containers (via pushContainer/popContainer)
-     */
-    export class ObjectEnumerationBuilder {
-        private instances: VisualObjectInstance[];
-        private containers: VisualObjectInstanceContainer[];
-        private containerIdx: number;
-
-        public pushInstance(instance: VisualObjectInstance, mergeInstances: boolean = true): ObjectEnumerationBuilder {
-
-            let instances = this.instances;
-            if (!instances) {
-                instances = this.instances = [];
-            }
+/**
+ * A helper class for building a VisualObjectInstanceEnumerationObject:
+ * - Allows call chaining (e.g., builder.pushInstance({...}).pushInstance({...})
+ * - Allows creating of containers (via pushContainer/popContainer)
+ */
+import * as _ from "lodash";
```

Comment:
do not import the standalone lodash library.
Install the required package instead.